tests/MockedPrecice.py

```diff
@@ -32,7 +39,7 @@ def is_action_required(self, py_action):
     def is_coupling_ongoing(self):
         raise Exception("not implemented")
 
-    def configure(self, foo):
+    def mark_action_fulfilled(self, py_action):
```

Comment:
`py_action` is misleading. It is not an action of the python action interface that preCICE offers, but simply an action like `action_write_initial_data` etc.

You could simply call it `action`.